packages/react-components/react-nav-preview/src/components/sharedNavStyles.styles.ts

```diff
@@ -42,6 +42,12 @@ export const useRootDefaultClassName = makeResetStyles({
   },
 });
 
+export const useSmallStyles = makeStyles({
+  root: {
+    padding: `${tokens.spacingVerticalXS} ${tokens.spacingVerticalS} ${tokens.spacingVerticalXS} ${tokens.spacingVerticalS}`,
```

Comment:
It looks like this works out to a 28px height to me with a quick check. Just want to make sure that's correct, since it's getting close to the WCAG 2.2 minimum target size requirement of 24px (though still passing! Just want to double check).

Reply:
scrubbed closely with design - to spec - thanks for the learnin' Sarah!